releasenotes/notes/run-async-azure-54450f0c2495f5c8.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add run async support for the AzureOpenAIChatGenerator
```

For consistency, I would make the release note a bit more elaborate.
```suggestion
---
---
features:
  - |
    Add `run_async` method to `AzureOpenAIChatGenerator`. This method uses `AsyncAzureOpenAI`
    to generate chat completions and supports the same parameters as the `run` method. It returns a coroutine
    that can be awaited.
```